deploy using tiup: align three PRs (#8603)
pingcap/docs: scale-tidb-using-tiup.md

## Scale out a TiDB/PD/TiKV cluster

> Assume that you want to add a TiDB node to the `10.0.1.5` host. Take the following steps.

Reviewer: 感觉修改前的版本更好

    To view the configuration of the current cluster, run `tiup cluster edit-config <cluster-name>`. Because the parameter configuration of `global` and `server_configs` is inherited by `scale-out.yaml` and thus also takes effect in `scale-out.yaml`.

2. Run the scale-out command:

    Before you run the `scale-out` command, use the `check` and `check --apply` commands to detect and automatically repair potential risks in the cluster:

    1. Check for potential risks:

        {{< copyable "shell-regular" >}}

        ```shell
        tiup cluster check <cluster-name> scale-out.yaml --cluster --user root [-p] [-i /home/root/.ssh/gcp_rsa]
        ```

    2. Enable automatic repair:

        {{< copyable "shell-regular" >}}

        ```shell
        tiup cluster check <cluster-name> scale-out.yaml --cluster --apply --user root [-p] [-i /home/root/.ssh/gcp_rsa]
        ```

    3. Run the `scale-out` command:

        {{< copyable "shell-regular" >}}

        ```shell
        tiup cluster scale-out <cluster-name> scale-out.yaml [-p] [-i /home/root/.ssh/gcp_rsa]
        ```

    In the preceding commands:

    - `scale-out.yaml` is the scale-out configuration file.

>     - `--user root` indicates logging in to the target machine as the root user to complete the cluster scale out. The root user should have`ssh` and `sudo` privileges to the target machine. Alternatively, you can use other users with `ssh` and `sudo` privileges to complete the deployment.

Reviewer: ditto

>     - `--user root` indicates logging in to the target machine as the root user to complete the cluster scale out. The root user should have`ssh` and `sudo` privileges to the target machine. Alternatively, you can use other users with `ssh` and `sudo` privileges to complete the deployment.

Author: ```suggestion
    - `--user root` indicates logging in to the target machine as the `root` user to complete the cluster scale out. The `root` user is expected to have `ssh` and `sudo` privileges to the target machine. Alternatively, you can use other users with `ssh` and `sudo` privileges to complete the deployment.
```

>     - `--user root` indicates logging in to the target machine as the root user to complete the cluster scale out. The root user should have`ssh` and `sudo` privileges to the target machine. Alternatively, you can use other users with `ssh` and `sudo` privileges to complete the deployment.

Author: ```suggestion
    - `--user root` indicates logging in to the target machine as the `root` user to complete the cluster scale out. The `root` user is expected to have `ssh` and `sudo` privileges to the target machine. Alternatively, you can use other users with `ssh` and `sudo` privileges to complete the deployment.
```

## Scale out a TiFlash cluster

> Assume that you want to add a TiFlash node to the `10.0.1.4` host. Take the following steps.

Reviewer: ditto, 个人倾向于原来的

## Scale in a TiFlash cluster

Assume that you want to remove a TiFlash node from the `10.0.1.4` host. Take the following steps.

### 1. Adjust the number of replicas of the tables according to the number of remaining TiFlash nodes

Before the node goes down, make sure that the number of remaining nodes in the TiFlash cluster is no smaller than the maximum number of replicas of all tables. Otherwise, modify the number of TiFlash replicas of the related tables.

1. For all tables whose replicas are greater than the number of remaining TiFlash nodes in the cluster, run the following command in the TiDB client:

    {{< copyable "sql" >}}

    ```sql
    ALTER TABLE <db-name>.<table-name> SET tiflash replica 0;
    ```

2. Wait for the TiFlash replicas of the related tables to be deleted. [Check the table replication progress](/tiflash/use-tiflash.md#check-replication-progress) and the replicas are deleted if the replication information of the related tables is not found.

### 2. Perform the scale-in operation

> perform the scale-in operation with one of the following solutions.

Reviewer: ```suggestion
Perform the scale-in operation with one of the following solutions.
```
